doc/Command-Reference.md

```diff
@@ -2348,6 +2348,74 @@ This command is used to delete a configured DHCP Relay Destination IP address or
   Restarting DHCP relay service...
   ```
 
+**config dhcp_relay ipv4 helper add/del**
+
+This command is used to add or delete an IPv4 DHCP Relay helper IP address or multiple IP addresses to a VLAN. Note that more than one DHCP Relay helper IP address can be operated on a VLAN interface.
+
+- Usage:
+  ```
+  config dhcp_relay ipv4 helper (add | del) <vlan_id> <dhcp_helper_ips>
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ sudo config dhcp_relay ipv4 helper add 1000 7.7.7.7
+  Added DHCP relay address [7.7.7.7] to Vlan1000
+  Restarting DHCP relay service...
+  ```
+
+  ```
+  admin@sonic:~$ sudo config dhcp_relay ipv4 helper add 1000 7.7.7.7 1.1.1.1
+  Added DHCP relay address [7.7.7.7, 1.1.1.1] to Vlan1000
+  Restarting DHCP relay service...
+  ```
+
+  ```
+  admin@sonic:~$ sudo config dhcp_relay ipv4 helper del 1000 7.7.7.7
+  Removed DHCP relay address [7.7.7.7] from Vlan1000
+  Restarting DHCP relay service...
+  ```
+
+  ```
+  admin@sonic:~$ sudo config dhcp_relay ipv4 helper del 1000 7.7.7.7 1.1.1.1
+  Removed DHCP relay address [7.7.7.7, 1.1.1.1] from Vlan1000
+  Restarting DHCP relay service...
+  ```
+
+**config dhcp_relay ipv6 destination add/del**
+
+This command is used to add or del an IPv6 DHCP Relay destination IP address or multiple IP addresses to a VLAN. Note that more than one DHCP Relay Destination IP address can be operated on a VLAN interface.
+
+- Usage:
+  ```
+  config dhcp_relay ipv6 destination (add | del) <vlan_id> <dhcp_destination_ips>
```

Comment:
Why destination and not helper?

Reply:
According to our discussion, add ipv4 helper and ipv6 destination. ipv4 helper part I added is in line2351-line2382

Reply:
Yes, helpers are DHCPv4 only concept, DHCPv6 it use relay destination.